### PR TITLE
use curlimages/curl debug container to run reorgs

### PR DESCRIPTION
**Pull request by skudasov**
Allow to use `kubectl debug` to spawn ephemeral containers and trigger reorgs on Geth

### CODE REVIEW

File: generate.go [hunk 1]
```diff
@@ -119,13 +119,14 @@ spec:
...
-	NodeInternalHTTPURL string    `yaml:"nodeInternalHTTPURL"`
-	Blocks              int64     `yaml:"blocks"`
+	ExperimentName        string    `yaml:"experimentName"`
+	Metadata              *Metadata `yaml:"metadata"`
+	Namespace             string    `yaml:"namespace"`
+	PodName               string    `yaml:"podName"`
+	ExecutorPodPrefix     string    `yaml:"executorPodPrefix"`
+	ExecutorContainerName string    `yaml:"executorContainerName"`
+	NodeInternalHTTPURL   string    `yaml:"nodeInternalHTTPURL"`
+	Blocks                int64     `yaml:"blocks"`
```
Comment (Tofel) [inline generate.go:129]: can we decide to use either snake or camelCase? this file is using camel, but `config.go` uses snake

Comment (skudasov) [reply generate.go:133]: TOML is using snake and YAML likes camel, I'm doing manifests so they look like ChaosMesh so camel-case there, and snake-case for TOML

File: testdata/snapshot/all/blockchain_rewind_head/blockchain_rewind_head-geth-7f7c9fb6c6-hzdhn-10.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+
+kind: blockchain_rewind_head
+name: blockchain_rewind_head-geth-7f7c9fb6c6-hzdhn-10
+metadata:
+  name: blockchain_rewind_head-geth-7f7c9fb6c6-hzdhn-10
+podName: geth-7f7c9fb6c6-hzdhn
+containerName: geth-network
+nodeInternalHTTPURL: geth:8544
+namespace: cl-cluster
+blocks: 10
```
Comment (Tofel) [inline testdata/snapshot/all/blockchain_rewind_head/blockchain_rewind_head-geth-7f7c9fb6c6-hzdhn-10.yaml:10]: is there a reason for committing these snapshots?

Comment (skudasov) [reply testdata/snapshot/all/blockchain_rewind_head/blockchain_rewind_head-geth-7f7c9fb6c6-hzdhn-10.yaml:10]: Yes, we are using them in tests

**Review by Tofel — APPROVED** [review]
LGTM!

**Pull request merged**